vignettes/NSSP_nowcast.Rmd

```diff
@@ -0,0 +1,539 @@
+Next we will make a plot of the nowcasted "final" cases compared to the initially reported, right-truncated cases and the true "final" reports.
+```{r}
+combined_data <- nowcast_w_data |>
+  select(reference_date, initial_count, final_count) |>
+  distinct() |>
+  pivot_longer(
+    cols = c(initial_count, final_count),
+    names_to = "type",
+    values_to = "count"
+  ) |>
+  mutate(type = case_when(
+    type == "initial_count" ~ "Initially observed data",
+    type == "final_count" ~ "Final observed data"
+  )) |>
+  filter(reference_date >= nowcast_date - days(60))
+```
+<details><summary>Click to expand code to create the plot of the probabilistic nowcast</summary>
+```{r}
+nowcast_data_recent <- nowcast_w_data |>
+  filter(reference_date >= nowcast_date - days(60))
+
+plot_prob_nowcast <- ggplot(nowcast_data_recent) +
+  geom_line(
+    aes(
+      x = reference_date, y = median
+    ),
+    color = "gray"
+  ) +
+  geom_ribbon(
+    aes(
+      x = reference_date,
+      ymin = q50th_lb, ymax = q50th_ub
+    ),
+    alpha = 0.5,
+    fill = "gray"
+  ) +
+  geom_ribbon(
+    aes(
+      x = reference_date,
+      ymin = q95th_lb, ymax = q95th_ub
+    ),
+    alpha = 0.5,
+    fill = "gray"
+  ) +
+  # Add observed data and final data once
+  geom_line(
+    data = combined_data,
+    aes(
+      x = reference_date,
+      y = count,
+      color = type
+    )
+  ) +
+  theme_bw() +
+  scale_color_manual(
+    values = c(
+      "Initially observed data" = "darkred",
+      "Final observed data" = "black"
+    ),
+    name = ""
+  ) +
+  xlab("Date of ED visit") +
+  ylab("Number of ARI cases") +
+  theme(legend.position = "bottom") +
+  ggtitle("Comparison of cases of ARI as of the nowcast date, later observed,\n and generated as a probabilistic nowcast") # nolint
+```
+</details>
+```{r}
+plot_prob_nowcast
+```
+
+# Summary
+
+In this vignette we used `baselinenowcast ` to nowcast cases of ARI starting from synthetic syndromic surveillance system data designed to mirror the U.S. NSSP dataset.
+We walked through the process of defining a syndromic surveillance definition using a list of diagnoses codes, and using the time stamps of updates in an electronic health record in the NSSP dataset to create a count of the number of cases of a specific definition indexed by the date of their visit and the date at which the patient's diagnoses was recorded into the surveillance system.
+From there we used the `baselinenowcast` workflow to estimate a delay distribution, apply it to generate a point nowcast, and estimate and apply uncertainty to generate probabilistic nowcasts.
+As a final step, we compared our nowcasts of the eventual final observed case counts to what we later observed and the right-truncated initial reports.
+
+Next steps include scoring the nowcasts we generated using proper scoring rules such as the weighted interval score or the continuous ranked probability score and found the coverage metrics.
```

Comment:
why are you teasing me give me some links it doesn't help me knowing you know how to do this I want to be able to do this.